e107_admin/image.php

```diff
@@ -195,7 +195,7 @@ function init()
 		);
 		
 		// FIXME LAN
-		$this->fields['media_cat_type']['writeParms'] = array('image' => 'Image', 'file' => 'File', 'video' => 'Video');
+		$this->fields['media_cat_type']['writeParms'] = array('image' => IMALAN_143, IMALAN_144 => 'File', 'video' => 'Video');
```

Comment:
changing a code value with a LAN will cause problems. For arrays in this "x => y" format, only "y" should ever be replaced with a LAN. 
`$this->fields['media_cat_type']['writeParms'] = array('image' =>IMALAN_143, 'file' => IMALAN_144, 'video' => OTHER_LAN_DEFINITION);`